x-pack/auditbeat/module/system/process/config.go

```diff
@@ -16,11 +17,19 @@ type Config struct {
 	ProcessStatePeriod time.Duration `config:"process.state.period"`
 
 	HasherConfig hasher.Config `config:"process.hash"`
+	Backend      string        `config:"process.backend"`
 }
 
 // Validate validates the config.
 func (c *Config) Validate() error {
-	return c.HasherConfig.Validate()
+	if err := c.HasherConfig.Validate(); err != nil {
+		return err
+	}
+	if c.Backend != "quark" && c.Backend != "proc" {
```

Comment:
We should be consistent with add_session_metadata in terms of `backend` option names.